redash/models.py

```diff
@@ -915,27 +931,33 @@ def outdated_queries(cls):
         return outdated_queries.values()
 
     @classmethod
-    def search(cls, term, group_ids, include_drafts=False):
-        # TODO: This is very naive implementation of search, to be replaced with PostgreSQL full-text-search solution.
-        where = (Query.name.ilike(u"%{}%".format(term)) |
-                 Query.description.ilike(u"%{}%".format(term)))
-
-        if term.isdigit():
-            where |= Query.id == term
-
-        where &= Query.is_archived == False
+    def search_queries(cls, term, group_ids, include_drafts=False, limit=20):
```

Comment:
Why append `_queries`? It's a method on the `Query` class isn't it obvious that it searches for queries? And in the handlers code you still call `.search`...

Reply:
Ugh, yeah, this is just a leftover from figuring out how to use the `search` method that the `SearchQueryMixin` adds to the `query_class` of the `Query` model. Will fix.